### PR TITLE
Updates to CRDs & Component Versions

### PR DESCRIPTION
**Pull request by andrewlecuyer**
Various CRD and component version updates.

[sc-13037]

### CODE REVIEW

File: helm/install/Chart.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: pgo
 description: A Helm chart for Kubernetes
 type: application
-version: 0.2.0
-appVersion: 5.0.3
+version: 0.3.0
```
Comment (jkatz) [inline helm/install/Chart.yaml:5]:
```suggestion
version: 0.2.1
```

File: helm/postgres/Chart.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: postgrescluster
 description: A Helm chart for Kubernetes
 type: application
-version: 0.2.0
-appVersion: 5.0.3
+version: 0.3.0
```
Comment (jkatz) [inline helm/postgres/Chart.yaml:5]:
```suggestion
version: 0.2.1
```

**Review by jkatz — CHANGES_REQUESTED** [review]

File: helm/install/Chart.yaml (hunk 1 above)
Comment (andrewlecuyer) [reply helm/install/Chart.yaml]: Updated the chart version to `0.2.1.`

File: helm/postgres/Chart.yaml (hunk 2 above)
Comment (andrewlecuyer) [reply helm/postgres/Chart.yaml]: Updated the chart version to `0.2.1.`

File: helm/install/values.yaml [hunk 3]
```diff
@@ -2,23 +2,23 @@
 ## Provide image repository and tag
 image:
   repository: registry.developers.crunchydata.com/crunchydata
-  tag: ubi8-5.0.3-0
+  tag: ubi8-5.0.4-0
```
Comment (jkatz) [inline helm/install/values.yaml:5]: See #57. Should be:

```yaml
image:
  image: registry.developers.crunchydata.com/crunchydata/postgres-operator:ubi8-5.0.4-0
```

**Review by jkatz — CHANGES_REQUESTED** [review]

File: helm/install/values.yaml (hunk 3 above)
Comment (andrewlecuyer) [reply helm/install/values.yaml]: Rebased to pull in change and then updated to `5.0.4`.

**Review by jkatz — APPROVED** [review]

**Pull request merged**